src/scss/components/intro.scss

```diff
@@ -0,0 +1,55 @@
+.intro {
+  background-image: url('../static/assets/blue-form.png');
+  background-position-x: right;
+  background-position-y: 254px;
+  background-repeat: no-repeat;
+
+  &__container-general {
+    align-items: center;
```

Comment:
when I inspect you component, this align moves the pictures in the wrong position.

![Captura de pantalla de 2020-08-13 22-08-56](https://user-images.githubusercontent.com/11698699/90209684-a1b27b00-ddb1-11ea-8a22-4f618d8320ce.png)
![Captura de pantalla de 2020-08-13 22-08-52](https://user-images.githubusercontent.com/11698699/90209722-bd1d8600-ddb1-11ea-867d-38c0972a3e20.png)

Reply:
I verify the bug with Camila and we show everthing normal in her pc, please check again and tell me if the issue persist.